src/store/modules/cosmos/bank/bank.js

```diff
@@ -0,0 +1,226 @@
+import Vue from 'vue'
+//import { Type, Field } from 'protobufjs' If registering custom message types
+const getDefaultState = () => {
+	return {
+		Balance: {},
+		AllBalances: {},
+		TotalSupply: {},
+		SupplyOf: {},
+		Params: {},
+		_Subscriptions: new Set()
```

Comment:
_Subscriptions is used internally by each auto-generated module. The store module registers an event listener for the env module's WS client newblock events (if WS is provided) and iterates over the set of subscriptions (set of actiontypes & params) to update the relevant parts of its store